src/components/Global/ProposalDropdown.vue

```diff
@@ -0,0 +1,35 @@
+<script setup>
+import { ref, watch, defineEmits } from 'vue'
+import { useUserDataStore } from '../../stores/userData'
+
+const emit = defineEmits(['selectionsComplete'])
+
+const userDataStore = useUserDataStore()
+const proposals = userDataStore.profile.proposals
+const activeProposals = proposals.filter(proposal => proposal.current === true)
+
+const selectedProposal = ref()
+
+watch(selectedProposal, (newValue) => {
+  emit('selectionsComplete', newValue)
+})
+
+</script>
+
+<template>
+  <template v-if="proposals.length">
+    <div>
+      <label for="proposalSelect">Select a proposal:</label>
+      <select id="proposalSelect" v-model="selectedProposal">
```

Comment:
rather than have a watcher on selectedProposal, I think its better to hook up the emit to the change/update event on the select component itself. Watchers are more hardcore and will watch any change vs the event will only be triggered when the component selects causing the change. This doesn't matter in this case, but could in cases where you might also programmatically change the value outside of that component (or in multiple components).

Reply:
good point! thanks Jon, I'll do that : )